peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -43,12 +43,17 @@ class Node(AbstractNode):
         window_loc (:obj:`Dict`): **default = { x: 0, y: 0 }** |br|
             X and Y coordinates of the top left corner of the displayed window,
             with reference from the top left corner of the screen, in pixels.
+
+    **See Also:**
+
+    :ref:`PeekingDuck Viewer<pkd_viewer>`: a graphical viewer for viewing
+    and studying the output of a PeekingDuck pipeline.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
-        cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
-        cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
+        # cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
+        # cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
```

Comment:
These were commented out to prevent an empty `opencv` window from being opened.
Will look into how to refine this portion.

Reply:
Went through the code base: there is currently no simple way to indicate to `output.screen` that `Viewer` is running so do not create the `opencv` `namedWindow`, because the data pool is not available at pipeline creation time, i.e. the instance when `output.screen` `__init__` is called.
Will need to think deeper into how to resolve this elegantly.

Reply:
Fix this by updating `DeclarativeLoader` to send a `pkd_viewer` flag via `config` to the node's `__init__`, and update `output.screen.__init__` to not create `opencv` `namedwindow` if `pkd_viewer` is `True`.